pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -96,8 +96,8 @@ func NewKubeGenericRuntimeManager(
 	cpuCFSQuota bool,
 	runtimeService internalApi.RuntimeService,
 	imageService internalApi.ImageManagerService,
-) (*kubeGenericRuntimeManager, error) {
-	kubeRuntimeManager := &kubeGenericRuntimeManager{
+) (*KubeGenericRuntimeManager, error) {
```

Comment:
You don't need to export `KubeGenericRuntimeManager`. Just return `kuberuntime.Runtime` here.